metricbeat/module/docker/image/data.go

```diff
@@ -0,0 +1,36 @@
+package image
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+
+	"github.com/elastic/beats/metricbeat/module/docker"
+	dc "github.com/fsouza/go-dockerclient"
+)
+
+func eventsMapping(imagesList []dc.APIImages) []common.MapStr {
+	events := []common.MapStr{}
+	for _, image := range imagesList {
+		events = append(events, eventMapping(&image))
+	}
+	return events
+}
+
+func eventMapping(image *dc.APIImages) common.MapStr {
+	event := common.MapStr{
+		"id": common.MapStr{
+			"current": image.ID,
+			"parent":  image.ParentID,
+		},
+		"created": image.Created,
+		"size": common.MapStr{
+			"regular": image.Size,
+			"virtual": image.VirtualSize,
+		},
+		"repoTags": image.RepoTags,
+	}
+	labels := docker.DeDotLabels(image.Labels)
```

Comment:
Oh, I see it has labels. Ignore my comment below. But it seems the line with the labels somehow breaks the tests. See Travis.